src/main/kotlin/com/deviceinsight/helm/PackageMojo.kt

```diff
@@ -29,7 +29,7 @@ import java.io.File
 class PackageMojo : AbstractHelmMojo() {
 
 	companion object {
-		private val PLACEHOLDER_REGEX = Regex("""\$\{(.*?)\}""")
+		private val PLACEHOLDER_REGEX = Regex("""\$\{(.*?)}""")
```

Comment:
I guess this was a unnecessarily escape?

Reply:
Exactly. Wouldn't have found it myself but IntelliJ was so kind to give me a hint. :wink: